tworaven_apps/eventdata_queries/dataverse/named_temporary_file.py

```diff
@@ -46,9 +46,8 @@ def __init__(self, query_json):
         # Do something (like send it to Dataverse)
```

Comment:
renamed this file as well as the class `NamedTemporaryFile`--e.g. it's already a python class